data/helpers.d/backend

```diff
@@ -40,7 +50,7 @@ $logfile {
 }
 EOF
 	sudo mkdir -p $(dirname "$logfile")	# Create the log directory, if not exist
-	cat ${app}-logrotate | sudo tee -a /etc/logrotate.d/$app > /dev/null	# Append this config to the others for this app. If a config file already exist
+	cat ${app}-logrotate | sudo $customtee /etc/logrotate.d/$app > /dev/null	# Append this config to the others for this app. If a config file already exist
```

Comment:
Here you must change the comment : "Append this config to the existing config file, or replace the whole config file (depending on $customtee)"